templates/items/details/details-facility.hbs

```diff
@@ -0,0 +1,63 @@
+<fieldset>
+    <legend>{{ localize "DND5E.ItemFacilityDetails" }}</legend>
+
+    {{!-- Type --}}
+    {{ formField fields.type.fields.value value=source.type.value choices=config.facilities.types labelAttr="label"
+                 blank=false }}
+
+    {{!-- Sub-Type --}}
+    {{ formField fields.type.fields.subtype value=source.type.subtype choices=facilitySubtypes }}
+
+    {{!-- Size --}}
+    {{ formField fields.size value=source.size choices=config.facilities.sizes }}
+
+    {{!-- Properties --}}
+    <div class="form-group split-group">
+        <label>{{ localize "DND5E.FACILITY.Properties" }}</label>
+        <div class="form-fields">
+
+            {{!-- Level --}}
+            {{ formField fields.level value=source.level label="DND5E.Level" localize=true classes="label-top" }}
+
+            {{!-- Order --}}
+            {{ formField fields.order value=source.order classes="label-top" choices=config.facilities.orders
+                         blank="" }}
```

Comment:
Feels like this might be better inside the Order section below, and just letting level live on its own line.